docs/sql-ref-syntax-hive-format.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: global
+title: Data Retrieval
+displayTitle: Data Retrieval
```

Comment:
In this case, shall we add a reference to this `sql-ref-syntax-hive-format.md` into `sql-ref-syntax-qry.md`?

Reply:
Yea, refer this in other menu doc is so strange and it's refer in two syntax doc with different type.